app/app.go

```diff
@@ -7,12 +7,17 @@ import (
 	"strings"
 )
 
-// extraArgs contains extra arguments to append to
-// os.Args. The arguments are separated with |.
-// Useful for running programs on mobiles where the
-// command line is not available.
-// Set with the go linker flag -X.
-var extraArgs string
+var (
+	// extraArgs contains extra arguments to append to
+	// os.Args. The arguments are separated with |.
+	// Useful for running programs on mobiles where the
+	// command line is not available.
+	// Set with the go linker flag -X.
+	extraArgs string
+	// ID Set with the go linker flag -X.
+	// Wayland: it sets the app_id
```

Comment:
This is difficult to understand for a Gio use that don't know the internals. How about

// ID is the app id exposed to the platform.
//
// On Android ID is the `package` property of AndroidManifest.xml,
// on iOS ID is the `CFBundleIdentifier` of the app Info.plist,
// on Wayland it's the ...
//
// ID is set by the `gogio` tool or manually with the -X linker flag:
//
//   EXAMPLE
//
// Note that ID is treated as a constant, and that changing it at runtime
// is not supported.

Note that the Android and IOS comments are a lie until `gogio` implements the linker flag.